spec/rubocop/lsp/server_spec.rb

```diff
@@ -1413,6 +1413,65 @@
     end
   end
 
+  describe 'didChange with multibyte character' do
```

Comment:
```suggestion
  describe '`textDocument/didChange` with multibyte character' do
```

Reply:
Hmm… The other tests are written as `did open` or `formatting`, so I’m thinking of changing it to `did open` to follow that pattern. What do you think?